server/index.js

```diff
@@ -6,20 +6,18 @@ const app = express();
 
 let port = process.env.port || 3002;
 
-const staticDirectory = path.join(__dirname, 'public');
+const staticDirectory = path.join(__dirname, '/../public/');
 const staticAssets = express.static(staticDirectory);
-app.use(staticAssets);
+app.use('/', staticAssets);
 
-app.get('/:offeringId/api/reviews', (req, res) => {
+app.get('/:offeringId', (req, res) => {
```

Comment:
Just a suggestion, but we might want to have our routes sitting behind something known and common such 
 as `/api` instead of `/` so that we don't pollute our root namespace. If we were building routes for the actual application that the users sees, it would make sense to have a given room page sitting at `/4` or `/offering/4`. However since we're pinging a backend API for data, most APIs tend to sit behind a specific API root route like `/api`.

Reply:
Just realized I may have confused you by what I said yesterday when you asked about how to set up routes, so my apologies if I'm being confusing!